e2e-tests/insurance/cypress/e2e/journeys/start.spec.js

```diff
@@ -84,11 +90,20 @@ context('Insurance Eligibility - start page', () => {
 
       cy.checkText(findOutMore.youCan(), FIND_OUT_MORE.YOU_CAN);
 
-      cy.checkLink(findOutMore.link(), FIND_OUT_MORE.LINK.HREF, FIND_OUT_MORE.LINK.TEXT);
+      cy.checkLink(findOutMore.link(), LINKS.EXTERNAL.GUIDANCE, FIND_OUT_MORE.LINK.TEXT);
```

Comment:
we could dereference LINKS.EXTERNAL on the top?

Reply:
Good shout, have updated.